build.sh

```diff
@@ -203,12 +232,27 @@ if [ -z "${TARGET_IMAGE:+x}" ]; then
 else
   read -ra TARGET_IMAGES <<<"-t $TARGET_IMAGE"
 fi
-if [ "$PS_VERSION" == "nightly" ]; then
-  ZIP_SOURCE="https://storage.googleapis.com/prestashop-core-nightly/nightly.zip"
-else
-  ZIP_SOURCE="https://github.com/PrestaShop/PrestaShop/releases/download/${PS_VERSION}/prestashop_${PS_VERSION}.zip"
+
+if [ -z "$ZIP_SOURCE" ]; then
```

Comment:
Better to override `ZIP_SOURCE` : if zip source is defined: use it. If not would default to nightly "https://storage.googleapis.com/prestashop-core-nightly/nightly.zip" or "https://github.com/PrestaShop/PrestaShop/releases/download/${PS_VERSION}/prestashop_${PS_VERSION}.zip"